UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceTests.cs

```diff
@@ -550,12 +613,12 @@ public async Task WhenInvalidWriteBlockInFileAsyncRequest_ThenReturnFalse()
                  Content = new StreamContent(new MemoryStream(Encoding.UTF8.GetBytes("Bad request")))
              });
 
-            var response = await fileShareService.UploadZipFileForExchangeSetToFileShareService(fakeBatchId, fakeExchangeSetPath, null);
-            Assert.AreEqual(false, response);
+            Assert.ThrowsAsync(Is.TypeOf<FulfilmentException>().And.Message.EqualTo("There has been a problem in creating your exchange set, so we are unable to fulfil your request at this time. Please contact UKHO Customer Services quoting error code : {0} and correlation ID : {1}"),
```

Comment:
Instead of adding message in each assert statement can we can one variable with that message and use that variable in all test cases.

Reply:
Resolved